projects/core/src/configurator/commons/facade/configurator-commons.service.ts

```diff
@@ -186,6 +186,22 @@ export class ConfiguratorCommonsService {
     );
   }
 
+  /**
+   * Verifies weather the configuration contains conflicts
+   *
+   * @param owner - Configuration owner
+   */
```

Comment:
Add **@return** and explain what is returned

Reply:
Done.